lib/elixir/lib/option_parser.ex

```diff
@@ -492,4 +556,34 @@ defmodule OptionParser do
       :unknown
     end
   end
+
+  defp format_errors(errors, opts) do
+    details = Enum.map_join(errors, ". ", &format_error(&1, opts))
+    total = Enum.count(errors)
+    error_pluralize = pluralize(total, "error", "errors")
```

Comment:
Yeah, that's true. Still not sure it's worth it here; in `ExUnit.CLIFormatter` it's used two times, so it's already extracting some common behaviour. Dunno, let's wait for other opinions maybe? :)